.github/release-drafter.yml

```diff
@@ -0,0 +1,4 @@
+template: |
+  ## Pull Requests
```

Comment:
This is just a bare bones template, I usually customize the text to include separate sections for "New features", "Breaking changes", "Documentation", "Library upgrades", etc.